### ISSUE TITLE
black throws an error when formatting string

### ISSUE THREAD

**Issue by dkuku** [psf/black#1607]
**Describe the bug** A clear and concise description of what the bug is.
black throws an error on my comment - 
**To Reproduce** Steps to reproduce the behavior:
```
class CITNowEvent(Resource):
    def post(self):
        """
        "event": "EVENTVIDEOPAGESERVED",  
        "server": "https://video.citnow.com",  
        "lid": "idev",  
        "lid_type": "service",  
        "dealername": "iDevices Test Site - DEALERSHIP NAME",  
        "dealerpostcode": "RG412PQ",  
        "timestamp": 1454508578 
        possible fields - the ones that we are interested
        _____________________________________
        | request | int    | Request | ID   |
        | video   | object | Video   | data |
        | vhc     | object | VHC     | data |
        """
```
thats the diff I get:
```
--- src
+++ dst
@@ -994,11 +994,11 @@
           body=
             Expr(
               value=
                 Constant(
                   value=
-                    '\n "event": "EVENTVIDEOPAGESERVED",  \n "server": "https://video.citnow.com",  \n "lid": "idev",  \n "lid_type": "service",  \n "dealername": "iDevices Test Site - DEALERSHIP NAME",  \n "dealerpostcode": "RG412PQ",  \n "timestamp": 1454508578 \n possible fields - the ones that we are interested\n _____________________________________\n | request | int    | Request | ID   |\n | video   | object | Video   | data |\n | vhc     | object | VHC     | data |\n ',  # str
+                    '\n "event": "EVENTVIDEOPAGESERVED",\n "server": "https://video.citnow.com",\n "lid": "idev",\n "lid_type": "service",\n "dealername": "iDevices Test Site - DEALERSHIP NAME",\n "dealerpostcode": "RG412PQ",\n "timestamp": 1454508578\n possible fields - the ones that we are interested\n _____________________________________\n | request | int    | Request | ID   |\n | video   | object | Video   | data |\n | vhc     | object | VHC     | data |\n ',  # str
                 )  # /Constant
             )  # /Expr
             Return(
               value=
                 Call(

```
After I remove it I can fromat the file, also it should not remove whitespaces from this string

**Comment by hugovk** [comment]
What error is being thrown?

Testing, Black 19.10b makes no changes:

```console
$ black --version
black, version 19.10b0
$ black 1607.py --check
All done! ✨ 🍰 ✨
1 file left unchanged.
```

And latest `master` runs without errors, but removes the spaces at the end of docstring lines:

```console
$ black --version
black, version 19.10b1.dev141+g820f387
$ black 1607.py
reformatted 1607.py
All done! ✨ 🍰 ✨
1 file reformatted.
```

![image](https://user-images.githubusercontent.com/1324225/90404675-64990200-e0ab-11ea-8ed6-4fd677f249a4.png)

Do you mean it shouldn't remove the spaces at the end of docstrings?

**Comment by dkuku** [comment]
I removed the whitespaces and it works ok but its enough to have one space at the end and it fails
I run it inside vim - the diff I pasted before, thats the error:
`INTERNAL ERROR: Black produced code that is not equivalent to the source.  Please report a bug on https://github.com/psf/black/issues.  This diff might be helpful: /tmp/blk_if5qr5x9.log `

**Comment by dkuku** [comment]
```
  ~/.config/nvim/.direnv/python-3.8.2/bin/black resources/citnow.py 
error: cannot format resources/citnow.py: INTERNAL ERROR: Black produced code that is not equivalent to the source.  Please report a bug on https://github.com/psf/black/issues.  This diff might be helpful: /tmp/blk_h6yjjoey.log
Oh no! 💥 💔 💥
1 file failed to reformat.

 ~/.config/nvim/.direnv/python-3.8.2/bin/black --version
black, version 19.10b1.dev90+g2432309
```

**Comment by hugovk** [comment]
I have version `19.10b1.dev141+g820f387`; that `devX` indicates something like `master` is now 51 commits ahead of you.

Please could you update your Black version to the newest `master` and retest?

**Comment by dkuku** [comment]
it works ok when I did pip -U black
false alarm

**Comment by JelleZijlstra** [comment]
This probably got fixed by #1593.

@dkuku `pip install -U black` would actually have downgraded black for you, since apparently you were previously on an unreleased commit, and the pip command would instead install the latest release (19.10b1). This bug was introduced and then fixed since that release.
